VERSION.txt

```diff
@@ -0,0 +1 @@
+0.0.1
```

Comment:
Would you mind renaming this file to `VERSION` to match what we do in our [other](https://github.com/aws/amazon-ecs-agent/blob/v1.17.3/VERSION) [projects](https://github.com/aws/amazon-ecs-cli/blob/v1.5.0/VERSION)?

Reply:
Fixed by c4d6021